mods/ra/mod.yaml

```diff
@@ -187,7 +187,7 @@ Fonts:
 		Font:./mods/common/FreeSansBold.ttf
 		Size:14
 	Title:
-		Font:./mods/ra/ZoodRangmah.ttf
+		Font:ra:ZoodRangmah.ttf
 		Size:48
 	MediumBold:
 		Font:./mods/common/FreeSansBold.ttf
```

Comment:
> I prefer consistency, having a mixed setup would confuse

or it would document the alternative way to set this up so it may stay as a test case and so we don't forget.